client/packages/dashboard/src/widgets/StockWidget.tsx

```diff
@@ -90,7 +92,10 @@ export const StockWidget: React.FC = () => {
                   value: formatNumber.round(expiryData?.expiringSoon),
                 },
               ]}
-            />            
+              link={RouteBuilder.create(AppRoute.Inventory)
+                .addPart(AppRoute.Stock)
```

Comment:
One thing missing from here which I forgot to add to the original PR is that these links don't quite make sense to me.

The closest part of the app which shows this information is stock. But maybe it would also make sense to add filters and/or sorting for expired or nearly expired stock.

Reply:
> But maybe it would also make sense to add filters and/or sorting for expired or nearly expired stock.

Yes, it does, and I was going to suggest that. But those lists don't currently support filtering by those fields, so it would be a bigger job to implement. Maybe make a new issue?

Reply:
I've made a new issue at #2269 

I wonder if I should leave out the links for these stock headers altogether at this stage. What do you think?

Reply:
I think it makes sense for the Widget headers to link to their respective areas, as you've done here.
At a later stage, we can add links to the sub-items (Expired batches, items not shipped, etc) to link to filtered views (in your new issue)

Reply:
We have a follow up issue for the filtered views but wanted to get this quick tweak done before doing any major UI changes to add filtering :) 
https://github.com/openmsupply/open-msupply/issues/2213